packages/mdc-chips/README.md

```diff
@@ -217,6 +219,9 @@ Method Signature | Description
 `deregisterTrailingIconInteractionHandler(evtType: string, handler: EventListener) => void` | Deregisters an event listener on the trailing icon element
 `notifyInteraction() => void` | Emits a custom event `MDCChip:interaction` denoting the chip has been interacted with
 `notifyTrailingIconInteraction() => void` | Emits a custom event `MDCChip:trailingIconInteraction` denoting the chip's trailing icon has been interacted with
+`notifyRemoval() => void` | Emits a custom event `MDCChip:removal` denoting the chip will be removed
```

Comment:
Is there a reason you preferred removal over remove (or removed) here?

Reply:
Just using a noun to mirror "interaction". Is "remove" more understandable?

Reply:
Oh, I guess that's true. I was thinking of e.g. "selected" on some other components. I guess removal works here.